app/support/date_formatter.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class DateFormatter
+  include ActionView::Helpers::DateHelper
```

Comment:
Just creating a regular Ruby class and including the module [`ActionView::Helpers::DateHelper`](http://api.rubyonrails.org/classes/ActionView/Helpers/DateHelper.html), which is a part of Rails.